src/html/tags/HtmlTagImg.js

```diff
@@ -15,24 +15,26 @@ const styles = StyleSheet.create({
 
 export default class HtmlTagImg extends React.PureComponent {
   props: {
-    src: string,
     auth: Auth,
     actions: Actions,
+    className: string,
     message: Message,
+    src: string,
     style: StyleObj,
   };
 
   handlePress = (resource: Object) => {
-    const { src, auth, message, actions } = this.props;
-    if (!isEmojiUrl(src, auth.realm)) {
+    const { className, src, auth, message, actions } = this.props;
+    if (!(className === 'emoji' || isEmojiUrl(src, auth.realm))) {
       actions.navigateToLightbox(resource, message);
     }
   };
 
   render() {
-    const { src, style, auth } = this.props;
+    const { className, src, style, auth } = this.props;
     const resource = getResource(src, auth);
-    const ContainerComponent = isEmojiUrl ? TouchableWithoutFeedback : Touchable;
+    const ContainerComponent =
+      className === 'emoji' || isEmojiUrl(src, auth.realm) ? TouchableWithoutFeedback : Touchable;
```

Comment:
What if we just replace it with a 'View' and then `handlePress` don't need to change. Even can be simplified.

Reply:
Also we can have a `const isEmoji className === 'emoji' || isEmojiUrl(src, auth.realm);` before the Container to simplify.